src/pyOpenMS/tests/unittests/test_AcquisitionInfo.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import os
+
+import pyopenms
+class TestAcquisitionInfo(unittest.TestCase):
+
+	def setUp(self):
+		dirname = os.path.dirname(os.path.abspath(__file__))
+		self.filename_mzml = os.path.join(dirname, "test.mzML").encode()
+
+	def test_acquisitioninfomemberaccess(self):
+		exp = pyopenms.MSExperiment()
+		pyopenms.MzMLFile().load(self.filename_mzml, exp)
+
+		# Basically test that the output is non-zero (e.g. the data is
```

Comment:
does the test file contain valid aquisition info? then I would also test for the content and not only if it is non-empty

Reply:
also I suggest testing iterators, push back etc

Reply:
Sure, can do. 
Since #4276 had no tests, I'll add them here.